src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -487,6 +486,25 @@ else if (null != sequenceDictionary) {
         }
     }
 
+
+    private void orderAndValidateAnnotations(final GATKReportTable annotationTable){
```

Comment:
Can you please add an integration test that fails before this fix, and passes after it? Also should add a basic unit test for the new `orderAndValidateAnnotations()` method.